sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -148,6 +162,14 @@ public SentryClient(final @NotNull SentryOptions options, @Nullable Connection c
       envelopeItems.add(sessionItem);
     }
 
+    if (attachments != null) {
+      for (Attachment attachment : attachments) {
```

Comment:
```suggestion
      for (final Attachment attachment : attachments) {
```

Reply:
Again manual (Manoel 😁) linter. Would be nice to have an automatic linter for that.

Reply:
Java is verbose, unless we write a custom lint, there's not one for this.
I like `val` in Kotlin, so that's why I use `final` in Java for everything, or do you use `var` for everything in Kotlin? :P